azureml-module-get-started.md

```diff
@@ -0,0 +1,76 @@
+
+   - The input/output data type (shown as port type in UI) is used for system to decide whether 2 ports can be connected in a pipeline. Port type can be any string without "<" and ">". If it's not "AnyFile" or "AnyDirectory", you need to register ([Python SDK](https://aka.ms/azureml-sdk-create-data-type)) the customized data type first before using it in module spec.
+   - AzureML service will use the parent folder of the yaml file to package as snapshot. If your source code and dependencies are not in the same folder with your yaml file, you can use field **[additionalIncludes](https://github.com/Azure/AzureML-Modules/blob/master/specs/topics/additional-includes.md)** to specify it.
+
+### Module registration and management
+
+AzureML provides command line interface ([Azure CLI](https://github.com/Azure/AzureML-Modules/blob/master/cli/doc.md)) for module registration and management. Refer to this [walk through](https://github.com/Azure/AzureML-Modules/blob/master/cli/walk-through.md) to register a module from GitHub repo, local folder or a zip package (typically created by a DevOps CI build job), update module to a new version and set its default version, disable/enable module, list and download module, etc.
+
+- “namespace” + “name” uniquely identifies a module in a workspace. So, make sure “version” is updated in the yaml every time you want to upgrade a module. 
+
+- Batch register – current module CLI supports one module per registration. If you want to batch register, you can leverage shell/batch/PowerShell script to iterate over directory. We leave this to our customers to allow more flexibility e.g. – you can decide if you want to use a specific naming pattern of the YAML files. An example to batch register the recommenders modules can be found in “Module Registration” section of this [tutorial](https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movieratings_with_azureml_designer.ipynb).
+
+In addition to Azure CLI, corresponding UX experience is provided for module management. Contact us <stcamlstudio@microsoft.com> to be involved in the preview of the experience.
+
+### Module consumption
+
+#### Python SDK
+
+A new Python SDK is provided to help you easily use custom modules to build a pipeline. Follow this [sample notebook](https://github.com/lisagreenview/hello-aml-modules/blob/master/pipeline_recommender.ipynb) to see how to build a SAR pipeline using previously registered modules. Full SDK reference document can be found here.
+
+- The input/output/parameter names called from the SDK should follow the Python language pattern, i.e. using “_” to replace space in the names.
+- _dsl.pipeline_ can be used to automatically infer modules used in a pipeline
```

Comment:
Use [inline code](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting) for `@dsl.pipeline` since it is code.

Reply:
I will remove this line as it will be addressed in the SDK reference doc.